src/lib/core/components/visualizations/implementations/HistogramVisualization.tsx

```diff
@@ -11,12 +11,15 @@ import {
   TimeDelta,
 } from '@veupathdb/components/lib/types/general';
 import { isTimeDelta } from '@veupathdb/components/lib/types/guards';
-import { HistogramData } from '@veupathdb/components/lib/types/plots';
+import {
+  HistogramData,
+  HistogramDataSeries,
+} from '@veupathdb/components/lib/types/plots';
 import { preorder } from '@veupathdb/wdk-client/lib/Utils/TreeUtils';
 import { getOrElse } from 'fp-ts/lib/Either';
 import { pipe } from 'fp-ts/lib/function';
 import * as t from 'io-ts';
-import { isEqual } from 'lodash';
+import { isEqual, sortBy, uniq, flatMap, max } from 'lodash';
```

Comment:
You'd need `zip` here, but no need for `flatMap` it's standard JS the way we use it.